workflows/create-pull-request.yaml

```diff
@@ -0,0 +1,59 @@
+name: Create Pull request
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    - cron: "44 0 * * *"
```

Comment:
it was an arbitrary random number.  The reason behind this was the load at full hours. It can be quite high, and cronjobs are delayed anyway. It is not very important, and I can change it back to 0 0 if it bothers your OCD ;)

Reply:
Actual cron also has a "delay by a random number of minutes" option - not sure if that is available here.

Reply:
I mean, if no one realizes that it's to avoid load at full hours, someone might well just randomly change it to a round number at some point not thinking much about it.

Reply:
@chriseth there is no such option (yet)
@cameel I will put a comment there